bin/git-scp

```diff
@@ -108,10 +108,10 @@ function scp_and_stage
 	then
 		local _TMP=${0///}
 		echo "$list" > "$_TMP" &&
-		_sanitize $list &&
+		_sanitize "$list" &&
 		_info "Pushing to $remote ($(git config "remote.$remote.url"))" &&
 		rsync -rlDv --files-from="$_TMP" ./ "$(git config "remote.$remote.url")/" &&
-		git add --force $list &&
+		git add --force "$list" &&
```

Comment:
These new changes will now cause `git add` to fail in all cases. Before, it would only fail on files that contained whitespace.